avocado/core/job.py

```diff
@@ -35,58 +34,12 @@
 from .job_id import create_unique_job_id
 from .future.settings import settings
 from .output import LOG_JOB, LOG_UI, STD_OUTPUT
-from .resolver import ReferenceResolutionResult, resolve
-from .tags import filter_test_tags_runnable
 from .test import DryRunTest
+from .resolver import resolve
 
 _NEW_ISSUE_LINK = 'https://github.com/avocado-framework/avocado/issues/new'
 
 
-def resolutions_to_tasks(resolutions, config):
```

Comment:
You moved `resolutions_to_tasks()` to `avocado.core.utils` here, but it's still referenced in `_make_test_suite_resolver()`.